code/modules/power/apc.dm

```diff
@@ -600,6 +600,9 @@
 			if (opened==2)
 				opened = 1
 			update_icon()
+	else if(istype(W, /obj/item/weapon/kitchen/utensil/fork) && opened) // Sticking fork in open APC shocks you
+		to_chat(user, "<span class='warning'>That was really, really dumb of you.</span>") // Why would you even do this
+		shock(user, 75, W.siemens_coefficient)
```

Comment:
This could be improved.

If connected to external power, shock based on amount of power in grid (like a powered grille does).
If not, small shock if it has a cell inserted.
Otherwise, no shock.